projects/dsp-ui/src/lib/viewer/values/interval-value/interval-input/interval-input.component.spec.ts

```diff
@@ -121,4 +150,45 @@ describe('InvertalInputComponent', () => {
 
     });
 
+    it('should mark the form\'s validity correctly', () => {
+        expect(testHostComponent.intervalInputComponent.valueRequiredValidator).toBe(true);
+        expect(testHostComponent.intervalInputComponent.form.valid).toBe(true);
+
+        testHostComponent.intervalInputComponent.startIntervalControl.setValue(null);
+
+        testHostComponent.intervalInputComponent._handleInput();
+
+        expect(testHostComponent.intervalInputComponent.form.valid).toBe(false);
+    });
+
```

Comment:
Could you add a test case for the `startEndSameTypeValidator`? Or is this not possible because the required operator will always trigger anyway (start or end is `null`)?

Reply:
done in 19f996d